molgenis-core-ui/src/main/resources/templates/molgenis-header.ftl

```diff
@@ -180,6 +180,9 @@
 <div class="row">
 <div class="col-md-12">
 <div id="plugin-container">
+    <#if plugin_description??>
```

Comment:
How to deal with long descriptions? See for example the data explorer entity type description and how we deal with abbreviation and hovering.